ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_creation.py

```diff
@@ -859,3 +859,48 @@ def test_jax_numpy_cdouble(
         on_device=on_device,
         x=x[0],
     )
+
+
+@handle_frontend_test(
+    fn_tree="jax.numpy.compress",
+    dtype_arr=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("integer"),
+        min_num_dims=2,
+        max_num_dims=3,
+        min_dim_size=10,
+        max_dim_size=100,
```

Comment:
You can use `dtype_values_axis` to generate the `axis` along with the array. In this way you won't have to limit the array's dimensions.